### ISSUE TITLE
Can't click and drag scrollbar for modal when scrollabe="true"

### ISSUE THREAD

**Issue by jlintag** [euvl/vue-js-modal#162]
Modal is passed in with properties:
```
<modal name="modal"
           height="auto"
           width="90%"
           :adaptive="true"
           :scrollable="true"
           @before-close="beforeClose">
      <sample-component>
      </sample-component>
</modal>
```
I can scroll fine with the mouse scroll wheel or the arrow keys just fine but clicking on the corresponding scrollbar for the modal closes the modal.

**Comment by euvl** [comment]
I dont think i can handle click events for scrollbars.

**Comment by JaZo** [comment]
@euvl, I don't agree with closing this as a question. In my opinion this is a bug. Fortunately @RoachMech came up with an easy fix, so I made PR #174 to solve this issue!

**Comment by amirshawn** [comment]
Thank you @JaZo 
This bug's been bothering me.
